### PR TITLE
nodePackages: Regenerate

### PR DESCRIPTION
**Pull request by the-kenny**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by Mic92** [comment]
Can you remove azure-cli? 

https://www.npmjs.com/package/azure-cli

It was deprecated by microsoft and no longer builds.

**Comment by Mic92** [comment]
Apart from that I have opened tickets for all broken packages.

**Comment by Ma27** [comment]
I'd also offer to have a look later tonight :)

**Comment by Mic92** [comment]
All the packages were that I mentioned were also broken before.

**Comment by Mic92** [comment]
@Ma27 `azure-cli` is deprecated by upstream. We can remove it as well.

**Comment by the-kenny** [comment]
I'll take a look if `azure-cli` works fine with `node8Packages`. Other than that, is there anything blocking this? :-)

**Comment by the-kenny** [comment]
Latest commit moves `azure-cli` to `node--packages-v8.json` and regenerates the package set.

**Comment by Ma27** [comment]
I pushed a commit which marks the remaining to packages as broken, it seems fine for me now.

> @Ma27 See: #36528 for azure-cli. I don't think we will able to get notified about security issues in our npm package set. That's why I would not include packages that are no longer upstream maintained.

Yeah, I've seen several tickets requesting the change as well, so I'm optimistic that this issue can be resolved soon. Regarding the package security issues: I agree for transitive dependencies deep inside the dependency tree, but this package doesn't seem to be dead yet and is the top-level package with a locked package set.

File: pkgs/development/node-packages/default-v10.nix [hunk 1]
```diff
@@ -90,4 +90,13 @@ nodePackages // {
     buildInputs = [ nodePackages.node-gyp-build ];
   };
 
+  # https://github.com/NixOS/nixpkgs/issues/51614
+  quassel-webserver = nodePackages.quassel-webserver.override {
```
Comment (Mic92) [inline pkgs/development/node-packages/default-v10.nix]: This one got removed now from nixpkgs and should be removed here as well.

File: pkgs/development/node-packages/default-v10.nix [hunk 2]
```diff
@@ -90,4 +90,13 @@ nodePackages // {
...
+    meta.broken = true;
+  };
+
+  # https://github.com/NixOS/nixpkgs/issues/51616
+  meguca = nodePackages.meguca.override {
```
Comment (Mic92) [inline pkgs/development/node-packages/default-v10.nix]: This one should be fixed now.

**Comment by Ma27** [comment]
ah great! In that case I'd revert my commit and regenerate the package set (a.k.a resolve the conflicts)

**Comment by Ma27** [comment]
@the-kenny FYI I rebased your changes onto master (where `meguca` got fixed and `quassel-webserver` got removed which resolves all merge conflicts btw).

To make the rebase easier, I squashed both regenerations into one commit and confirmed that it works by running `nix-review rev 6d21e8d361157697dfe870f90eddceacecc3e576`. I'll wait for ofBorg to evaluate, then this is mergable IMHO.

**Comment by Ma27** [comment]
@the-kenny thanks!

**Comment by the-kenny** [comment]
Awesome! Thank you all :-)

**Pull request merged**